Fix up Sightings list to be in reverse order with most recent ones first

morejanus opened the pull request.

Commit:
🍮🏬 Checkpoint
./detail.js:14193219/76

Commit:
Merge branch 'master' of https://github.com/morejanus/manta-ray-id-project

Commit:
Import of morejanus/manta-ray-id-project

Commit:
🙇🕌 Checkpoint
./detail.js:14193219/34

Dthurow commented on detail.js:
```diff
@@ -85,8 +85,8 @@ var app = new Vue({
                         this.sightings = data.data.sort(function(a,b){
-                            return new Date(b.StartDateTime) - new Date(a.StartDateTime);
-                          });
+                            return (a.StartDateTime - b.StartDateTime);
+                          }).reverse();
```
For some reason my brain is saying this is the same thing as the original sort function. Do you have an example detail page where it was loading in the wrong order so I can work thru an example where the original sort doesn't work? I was using queenie as my example manta ray, and it works for that one.

the original sort was b-a, and the new one is a-b, then reverse it.

morejanus replied:
I thought that b-a would work as well but when I tried it, it didn't. I also tried by converting the string to a Date as you did, and that didn't work for me either. I used #9 Big Bertha as my test but also just used Queenie and it's always in ascending order. I just updated the website with the original code to double check, and it's in ascending order.

morejanus committed:
Resolved merge conflict

morejanus committed:
-Add new about.html file and have other pages link to it
-Turn off caching when requesting Photo Objects and Sighting information on the detail page
-Make links viewable as such in basic text
